features/cells/src/main/java/com/wire/android/feature/cells/ui/common/LoadingScreen.kt

```diff
@@ -17,27 +17,79 @@
  */
 package com.wire.android.feature.cells.ui.common
 
+import androidx.compose.foundation.border
 import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.foundation.shape.CircleShape
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.Alignment
+import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.draw.clip
+import com.wire.android.model.Clickable
 import com.wire.android.ui.common.colorsScheme
 import com.wire.android.ui.common.dimensions
-import com.wire.android.ui.common.progress.WireCircularProgressIndicator
+import com.wire.android.ui.common.rowitem.RowItemTemplate
+import com.wire.android.ui.common.shimmerPlaceholder
 
 @Composable
 fun LoadingScreen(
```

Comment:
Maybe we could move this `LoadingScreen` to the common module as well - it uses `RowItemTemplate` inside anyway, not any specific item composable, so maybe we could be able to reuse this `LoadingScreen` in some other places where `RowItemTemplate` is used 🤔